ansible/roles/operator-pipeline/templates/openshift/triggers/operator-hosted-pipeline-trigger.yml

```diff
@@ -0,0 +1,138 @@
+# Other GitHub PR webhook payload example: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: TriggerBinding
+metadata:
+  name: operator-hosted-pipeline-trigger-binding
+spec:
+  params:
+    - name: git_pr_branch
+      value: $(body.pull_request.head.ref)
+    - name: git_pr_title
+      value: $(body.pull_request.title)
+    - name: git_pr_url
+      value: $(body.pull_request.html_url)
+    - name: git_fork_url
+      value: $(body.pull_request.head.repo.html_url)
+    - name: git_repo_url
+      value: $(body.pull_request.base.repo.html_url)
+    - name: git_username
+      value: $(body.pull_request.head.user.login)
+    - name: pr_head_label
+      value: $(body.pull_request.head.label)
+    - name: pyxis_url
+      value: https://pyxis.engineering.redhat.com/
+    - name: preflight_min_version
+      value: "1.0"
+    - name: ci_min_version
+      value: "1.0"
+
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: TriggerTemplate
+metadata:
+  name: operator-hosted-pipeline-trigger-template
+spec:
+  params:
+    - name: git_pr_branch
+    - name: git_pr_title
+    - name: git_pr_url
+    - name: git_fork_url
+    - name: git_repo_url
+    - name: git_username
+    - name: pr_head_label
+    - name: bundle_path
+    - name: pyxis_url
+    - name: preflight_min_version
+    - name: ci_min_version
+  resourcetemplates:
+    - apiVersion: tekton.dev/v1beta1
+      kind: PipelineRun
+      metadata:
+        generateName: operator-hosted-pipeline-run-
+      spec:
+        pipelineRef:
+          name: operator-hosted-pipeline
+        params:
+          - name: git_pr_branch
+            value: $(tt.params.git_pr_branch)
+          - name: git_pr_title
+            value: $(tt.params.git_pr_title)
+          - name: git_pr_url
+            value: $(tt.params.git_pr_url)
+          - name: git_fork_url
+            value: $(tt.params.git_fork_url)
+          - name: git_repo_url
+            value: $(tt.params.git_repo_url)
+          - name: git_username
+            value: $(tt.params.git_username)
+          - name: pr_head_label
+            value: $(tt.params.pr_head_label)
+          - name: bundle_path
+            value: $(tt.params.bundle_path)
+          - name: pyxis_url
+            value: $(tt.params.pyxis_url)
+          - name: preflight_min_version
+            value: $(tt.params.preflight_min_version)
+          - name: ci_min_version
+            value: $(tt.params.ci_min_version)
+        workspaces:
+          - name: repository
+            volumeClaimTemplate: &volumeClaimTemplate5Gi
+              spec:
+                accessModes:
+                  - ReadWriteOnce
+                resources:
+                  requests:
+                    storage: 5Gi
+          - name: results
+            volumeClaimTemplate: *volumeClaimTemplate5Gi
```

Comment:
5GB is probably overkill for the results. How do you feel about reducing this to 100MB? It's still a very large amount of data for a single certification run and it'll all be uploaded to pyxis. Reducing this claim is a way to keep some reasonable limit in place.